l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -5,15 +5,17 @@
 from .labels import LABELS
 
 # When changing the schema bump this number
-FORMAT_VERSION = 1
+FORMAT_VERSION = 2
 
 FRAME_ARRAY_KEY = "frames"
 AGENT_ARRAY_KEY = "agents"
 SCENE_ARRAY_KEY = "scenes"
+TR_FACES_ARRAY_KEY = "traffic_faces"
 
 FRAME_CHUNK_SIZE = (10_000,)
 AGENT_CHUNK_SIZE = (20_000,)
 SCENE_CHUNK_SIZE = (10_000,)
+TR_FACES_CHUNK_SIZE = (10_000,)
```

Comment:
I would not use too short names here, `TR` is not very telling. Prefer `TRAFFIC_LIGHTS_CHUNK_SIZE`